packages/server/src/auth/auth.service.ts

```diff
@@ -147,4 +149,24 @@ export class AuthService {
     publicKeys.push(this.configService.get('PUBLIC_KEY_2'));
     return publicKeys;
   }
+
+  /**
+   * Get list of projects that a user has access to and is an admin of
+   *
+   * @returns List of projects
+   */
+  async projects(userId: string): Promise<Project[]> {
+    const firstUser = await this.userService.findUserById(userId);
+    const email = firstUser.email;
+
+    const users = await this.userService.findUsersByEmail(email);
+    const projects = [];
+    for (const user of users) {
+      if (user.role == 1) {
```

Comment:
```suggestion
      if (user.role === Roles.Admin) {
```
We should use the Role enum as much as possible so we don't have to comment on why the number is 1 here.